Rakefile

```diff
@@ -0,0 +1,76 @@
+#
+# Cookbook Name:: python-webapp
+#
+# Copyright Chef, OSU Open Source Lab
+#
+# Original authors: https://github.com/chef-cookbooks/yum/blob/master/Rakefile
+#
+# Licensed under the Apache License, Version 2.0 (the 'License');
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an 'AS IS' BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require 'rspec/core/rake_task'
+require 'rubocop/rake_task'
+require 'foodcritic'
+require 'kitchen'
+
+# Style tests. Rubocop and Foodcritic
+namespace :style do
+  desc 'Run Ruby style checks'
+  RuboCop::RakeTask.new(:ruby)
+
+  desc 'Run Chef style checks'
+  FoodCritic::Rake::LintTask.new(:chef) do |t|
+    t.options = {
+      fail_tags: ['any']
+    }
+  end
+end
+
+desc 'Run all style checks'
+task style: %w(style:chef style:ruby)
+
+# Rspec and ChefSpec
+desc 'Run ChefSpec tests'
+RSpec::Core::RakeTask.new(:spec)
+
+# Integration tests. Kitchen.ci
+namespace :integration do
+  desc 'Run Test Kitchen with Vagrant'
+  task :vagrant do
+    Kitchen.logger = Kitchen.default_file_logger
+    Kitchen::Config.new.instances.each do |instance|
+      instance.test(:always)
+    end
+  end
+
+  desc 'Run Test Kitchen with cloud plugins'
+  task :cloud do
+    Kitchen.logger = Kitchen.default_file_logger
+    @loader = Kitchen::Loader::YAML.new(project_config: './.kitchen.cloud.yml')
+    config = Kitchen::Config.new(loader: @loader)
+    config.instances.each do |instance|
+      instance.test(:always)
+    end
+  end
+end
+
+desc 'Run all tests (style, spec, integration) on Openstack'
+task cloud: %w(style spec integration:cloud)
+
+desc 'Run all tests (style, spec, integration) on Vagrant'
+task vagrant: %w(style spec integration:vagrant)
```

Comment:
This is the default task, as documented in the README. I suppose it could've had a better 'desc', or one at all.